config/deploy.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+# config valid for current version and patch releases of Capistrano
+lock '~> 3.17.1'
+require File.expand_path('./environment', __dir__)
+
+set :application, 'bpldc_authority_api'
+set :repo_url, "https://github.com/boston-library/#{fetch(:application)}.git"
+## Make user home path dynamic..
+set :user, Rails.application.credentials.dig(:deploy, :user)
+set :deploy_to, "/home/#{fetch(:user)}/#{fetch(:application)}"
+
+set :rvm_installed, '/home/manager/.rvm/bin/rvm'
```

Comment:
Instead of hard-coding this, there should be a way to get the path to the rvm path from the environment.

Reply:
Since the rvm path is always under ~/.rvm/bin/rvm, use #{fetch(:user)} for absolute rvm path